### DISCUSSION TITLE
寒霜弹幕使引擎在大量弹幕的视频中会使弹幕重叠

### DISCUSSION THREAD

**Discussion by a0iwguf09u** [ywmoyue/biliuwp-lite#388] [bug]
### Bug 描述

在选择设置中的寒霜弹幕使引擎之后,打开弹幕较多的视频,弹幕会发生重叠,无论怎么调节都没用

### 操作步骤

```plain text
1.打开应用
2.点击右上方设置
3.在"视频弹幕"选项卡中,将"搜索引擎"切换至"寒霜弹幕使"
4.随机打开一个弹幕多的视频
5.出现弹幕重叠情况
```

### 期望结果

弹幕不再重叠

### 截图

![image](https://github.com/ywmoyue/biliuwp-lite/assets/118047201/012611a4-e508-4801-99ed-297af1b08670)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/118047201/cd887d70-7e84-468a-8872-a4f6f799da4e)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/118047201/52581ff8-5f1d-41df-b4c3-c374157e2ab2)

### 日志

[20231031.log](https://github.com/ywmoyue/biliuwp-lite/files/13218437/20231031.log)

### 已定位描述

还没发现

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 专业版22H2 19045.3570

### 设备类型

台式机

### 设备架构

x64

### 应用架构

x64

### 备注

好像没了

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
在弹幕密度过高时，堆叠是不可避免的。 毕竟屏幕空间就那么大，如果弹幕已经布满屏幕那势必要堆叠起来。
https://github.com/ywmoyue/biliuwp-lite/discussions/13#discussioncomment-7215416 NSDanmaku 在设计时就考虑了不堆叠而去除多余的弹幕。 可以先使用NSDanmaku.
